### DISCUSSION TITLE
Unable to reach AdGuard WebUI

### DISCUSSION THREAD

**Discussion by jokerigno** [tteck/Proxmox#2014] [Help Needed]
Hi,

I installed this LXC some months ago and worked flawlessy till today when I noticed that webgui is no longer reachable. If I try with the defined IP (192.168.2.103) I receive the ERR_CONNECTION_REFUSED message (tried with edge and firefox.

Any advice?

**Comment by tteck** [comment]
What's the output from `hostname -I`?

**Comment by jokerigno** [comment]
that' exactly the same IP adress:

root@adguard:~# hostname -I
192.168.2.103 

![immagine](https://github.com/tteck/Proxmox/assets/23552762/97b3d137-ae9a-44d8-b9e7-823bc096add9)

**Reply by X-Anonymous-Y** [reply]
Are you use a VPN? It was for me the same error yesterday with Wiregueard VPN

**Reply by jokerigno** [reply]
I'm running a Tailgate instance inside Home Assistant VM (that is running as an addon). But this worked without issue. Maybe something has broken recently? What do you suggest?

**Reply by X-Anonymous-Y** [reply]
I just created a new VPN tunnel and that worked for me.

**Comment by davethecripple** [comment]
Were you ever able to resolve this? I'm running into the same exact issue - unable to reach the WebUI - worked flawlessly up until 2 weeks ago - IP address matches still.

**Reply by tteck** [reply]
What does `systemctl status AdGuardHome` show?

**Reply by jokerigno** [reply]
nope. I recreated the container and now it seem working again.

**Reply by neyzm** [reply]
I'm having the same issue, Running `systemctl` seems fine, but when I run `journalctl` it shows the following:

```-- Boot 923f6cdee18c4a21bd370ebb02bea5b2 --
Feb 03 20:15:45 adguard systemd[1]: Starting AdGuard Home: Network-level blocker...
Feb 03 20:15:45 adguard systemd[1]: Started AdGuard Home: Network-level blocker.
Feb 04 12:40:09 adguard systemd[1]: Stopping AdGuard Home: Network-level blocker...
Feb 04 12:40:09 adguard systemd[1]: AdGuardHome.service: Succeeded.
Feb 04 12:40:09 adguard systemd[1]: Stopped AdGuard Home: Network-level blocker.
Feb 04 12:40:09 adguard systemd[1]: AdGuardHome.service: Consumed 58.075s CPU time.```
